data/jobs.txt

```diff
@@ -1573,3 +1573,192 @@ mission "Large Rush Delivery [3]"
 		payment
 		payment 42000
 		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+mission "Dump garbage"
```

Comment:
Heh. I guess I could make it more about recycling, which would explain why you have to dump it on a planet with a giant incinerator.